Ghidra/Processors/SuperH/data/languages/superh.sinc

```diff
@@ -1021,74 +1021,25 @@ MovMUReg2: MovMUReg2_15 is MovMUReg2_15 {
     local tmp1:1;
     local tmp2:4;
     local old_q:1;
+    local old_q_eq_m:1;
```

Comment:
Not sure why local vars are pre-declared (although this had previously been done this way).  I generally find declaration on first write to be easier to read. (OPTIONAL)